frontend/src/types/AuthTypes.ts

```diff
@@ -5,6 +5,11 @@ export enum TabEnum {
   TEACHER,
 }
 
+export enum TabEnumClassroom {
```

Comment:
nit: might be better to move this to another file? It's not really relevant to "auth"